src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/default-extension.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['services/husky/util', 'services/husky/translator'], function(util, translator) {
+
+    'use strict';
+
+    return function(app) {
+        /**
+         * Gets executed every time BEFORE a component gets initialized.
+         * Loads data if needed and start executing component handlers
+         */
+        app.components.before('initialize', function() {
+            if (!this.defaults) {
+                return;
+            }
+
+            // merge options
+            if (!!this.defaults.options) {
+                this.options = util.extend(true, {}, this.defaults.options, this.options);
+            }
+
+            // merge translations and translate the values
+            if (!!this.defaults.translations) {
+                var translations = util.extend(true, {}, this.defaults.translations, this.options.translations || {});
+                this.translations = util.object(
+                    util.arrayMap(translations, function(item, key) {
+                        return [key, translator.translate(item)];
+                    })
+                );
+            }
+
+            // merge templates and prepare template functions
+            if (!!this.defaults.templates) {
+                var templates = util.extend(true, {}, this.defaults.templates, this.options.templates || {});
+                this.templates = util.object(
```

Comment:
we could introduce default options with templates and translations! would be cool for cascading templates!